TECHMANIA/Assets/Scripts/Paths.cs

```diff
@@ -269,6 +269,24 @@ public static List<string> GetAllVideoFiles(string folder)
 
     public static string FullPathToUri(string fullPath)
     {
+#if UNITY_ANDROID
+        if (fullPath.Contains(Application.streamingAssetsPath))
+        {
+            return fullPath;
```

Comment:
`// Streaming assets on Android are not files, so they are inaccessible with "file://".`